### PR TITLE
Fix disk in partitions

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: test/mbr_update_test.rb [hunk 1]
```diff
@@ -134,9 +134,19 @@ def stage1(devices: [], activate: false, generic_mbr: false)
           expect(Yast::Execute).to receive(:locally)
             .with(/parted/, "-s", "/dev/sda", "set", 1, "boot", "on")
           expect(Yast::Execute).to receive(:locally)
-            .with(/parted/, "-s", "/dev/sdb", "set", 1, "boot", "on")
+            .with(/parted/, "-s", "/dev/sdb", "set", 2, "boot", "on")
 
-          subject.run(stage1(activate: true, devices: ["/dev/sda1", "/dev/sdb1"]))
+          subject.run(stage1(activate: true, devices: ["/dev/sda1", "/dev/sdb2"]))
```
Comment (ancorgs) [inline test/mbr_update_test.rb]: Any reason for this change from sdb1 to sdb2? Just curious

File: package/yast2-bootloader.changes [hunk 2]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Jan 16 16:51:54 UTC 2018 - jreidinger@suse.com
+
+- fix crash for getting partition to activate when stage1 device
+  is disk (bsc#10761210
```
Comment (ancorgs) [inline package/yast2-bootloader.changes]: Matching closing parenthesis not found. Buffer overflow. ;-)

File: test/mbr_update_test.rb (hunk 1 above)
Comment (jreidinger) [reply test/mbr_update_test.rb]: well, for partition it just use any partition which is in stage1, but it does not make any sense to have stage1 in swap partition.

Comment (jreidinger) [reply test/mbr_update_test.rb]: so it is just to have reasonable test cases

Comment (ancorgs) [reply test/mbr_update_test.rb]: Oh, is explained below

File: package/yast2-bootloader.changes [hunk 3]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Jan 16 16:51:54 UTC 2018 - jreidinger@suse.com
+
+- fix crash for getting partition to activate when stage1 device
+  is disk (bsc#10761210)
```
Comment (ancorgs) [inline package/yast2-bootloader.changes]: Bug #10761210 does not exist, according to bugzilla.suse.com. Wrong copy&paste?

Comment (ancorgs) [reply package/yast2-bootloader.changes]: Yes, you wrote an extra 0 at the end.

File: test/mbr_update_test.rb (hunk 1 above)
Comment (jreidinger) [reply test/mbr_update_test.rb]: yeah, I do not notice it first time. I just use sdb1 in other test case and was surprised why sdb2 was picked and then inspect partitioning which makes it clear :)

File: package/yast2-bootloader.changes (hunk 3 above)
Comment (jreidinger) [reply package/yast2-bootloader.changes]: ah, typo. I use 0 instead of ) and appending ) does not fix it :)

**Pull request merged**